rotkehlchen/chain/ethereum/modules/nfts.py

```diff
@@ -221,44 +221,52 @@ def get_balances(
 
         return result
 
-    def get_nfts_with_price(self) -> List[Dict[str, Any]]:
-        cursor = self.db.conn.cursor()
-        query = cursor.execute('SELECT identifier, last_price, last_price_asset, manual_price from nfts WHERE last_price IS NOT NULL')  # noqa: E501
-        result = []
-        for entry in query:
-            to_asset_id = entry[2] if entry[2] is not None else A_USD
-            try:
-                to_asset = Asset(to_asset_id)
-            except UnknownAsset:
-                log.error(f'Unknown asset {to_asset_id} in custom nft price DB table. Ignoring.')
-                continue
+    def get_nfts_with_price(self, identifier: Optional[str] = None) -> List[Dict[str, Any]]:
```

Comment:
add a docstring

Reply:
This is not new code. Not sure why github shows it in the diff like this :/

Reply:
because code moved. Just add the docstring